### ISSUE TITLE
Why not use more channels to predict?

### ISSUE THREAD

**Issue by qianxiaokang** [perslev/U-Time#11]
Hi Perslev

I read your paper and found that you only used 2 or 3 channels to predict a label at most. But the rule of AASM is to predict a label according to 7 channels. If more channels are used, will the accuracy be improved?

Thanks

**Comment by perslev** [comment]
Hi qianxiaokang,

Sorry for my late reply. It definitely might make sense to include additional channels in many cases. However, in doing so the target function that your model is to approximate also becomes more complex, which means that you will most likely need additional training data to fit a good model.

In other words, if you have a large training dataset available, then I would definitely suggest you to use additional input channels, and I would expect the accuracy to be improved. For smaller datasets, as we show in the paper, you can often reach reasonable to good performance even with just a single EEG channel. Adding a single EOG often improves the performance further.

I unfortunately cannot give an exact estimate of how much data you need "per input channel". That I will leave you to experiment with :)

Please let me know if you have further questions.

Mathias

**Comment by qianxiaokang** [comment]
Thank you for your explanations.
